test/integration/epp/hermetic_test.go

```diff
@@ -98,7 +98,7 @@ func TestFullDuplexStreamed_KubeInferenceModelRequest(t *testing.T) {
 	tests := []struct {
 		name              string
 		requests          []*extProcPb.ProcessingRequest
-		pods              map[backend.Pod]*backendmetrics.Metrics
+		pods              map[*backend.Pod]*backendmetrics.Metrics
```

Comment:
once labels map was added into `backend.Pod`, it's not possible to use it as key in map. therefore changed to pointer (which anyway seems fine).